backend/src/main/java/com/empire/World.java

```diff
@@ -1393,7 +1387,7 @@ void spawnPirates() {
 				}
 			}
 			for (String bribe : pirate.bribes.keySet()) pirate.bribes.put(bribe, pirate.bribes.get(bribe) * .75);
-			notifyAllPlayers("Piracy", (int) Math.ceil(piratesSpawned) + " total pirates have appeared in " + StringUtil.and(pirateNotes) + ".");
+			notifyAllPlayers("Piracy", (int) Math.ceil(piratesSpawned) + " total pirates have appeared in " + StringUtil.and(new ArrayList<>(pirateNotes)) + ".");
```

Comment:
You could avoid the extra Set -> List conversion when using and() by changing and() to accept a Collection<> instead of a List<>

Reply:
Good point! Done.